tiled/client/awkward.py

```diff
@@ -0,0 +1,52 @@
+import awkward
+
+from ..serialization.awkward import from_zipped_buffers, to_zipped_buffers
+from ..structures.awkward import project_form
+from .base import BaseClient
+from .utils import handle_error
+
+
+class AwkwardArrayClient(BaseClient):
+    def __repr__(self):
+        # TODO Include some summary of the structure. Probably
+        # lift __repr__ code from awkward itself here.
+        return f"<{type(self).__name__}>"
+
+    def write(self, container):
+        handle_error(
+            self.context.http_client.put(
+                self.item["links"]["full"],
+                content=bytes(to_zipped_buffers(container, {})),
+                headers={"Content-Type": "application/zip"},
+            )
+        )
+
+    def read(self, slice=...):
+        structure = self.structure()
+        form = awkward.forms.from_dict(structure.form)
+        typetracer, report = awkward.typetracer.typetracer_with_report(
+            form,
+            forget_length=True,
```

Comment:
This argument is default `True`, and seeing it used here made me realise that we [probably want to deprecate it](https://github.com/scikit-hep/awkward/pull/2671). Could you remove the usage?
```suggestion
```